pixi.toml

```diff
@@ -33,10 +33,16 @@ py-test = { cmd = "python -m pytest -vv rerun_py/tests/unit", depends_on = [
   "py-build",
 ] }
 
-cpp-prepare = "cmake -G Ninja -B build -S . -DCMAKE_BUILD_TYPE=Debug"
+cpp-prepare = "cmake -G Ninja -B build -S . -DCMAKE_BUILD_TYPE=Debug -DCMAKE_COMPILE_WARNING_AS_ERROR=ON"
```

Comment:
I'm curious how much running this every time slows down the dev-cycle. Should we consider making this a no-op if the `build` folder already exists?

Reply:
Hmm ideally I'd like to parameterize this to make it opt-in. CI should always opt-in

Reply:
I’ve documented how to do that in the pixi.toml file

Reply:
thanks, and that's nice and easy. But there doesn't seem to be a way to pass things through to the `depends_on` part? I'd like to be able to `cpp-build -DCMAKE_COMPILE_WARNING_AS_ERROR=ON` which would forward this argument to the underlying `cpp-prepare` 🤔

Reply:
Ask on the pixi Discord :)

Reply:
env vars is the way to go for now it seems!